generator/config/expressions.php

```diff
@@ -83,11 +83,11 @@
     ],
     'fieldQuery' => [
         'returnType' => Type\FieldQueryInterface::class,
-        'acceptedTypes' => [Type\FieldQueryInterface::class, ...$bsonTypes['any']],
+        'acceptedTypes' => [Type\FieldQueryInterface::class, BSON\Regex::class, 'bool', 'string', 'array', 'null', stdClass::class, ...$bsonTypes['number']],
```

Comment:
Why is Regex specifically allowed but not other special BSON types? Does everything else require use of `eq()` or some other comparison operator?

Reply:
Reverted in https://github.com/mongodb/mongo-php-builder/pull/11